simulator/mips/mips_instr.cpp

```diff
@@ -231,6 +256,20 @@ void FuncInstr::initFormat()
             return;
         }
     }
+    
+    else
+    {
+        const auto& entry = isaMapIJ.at( ident);
+        if ( entry != isaMapIJ.end())
+        {
+            format    = entry.format;
+            operation = entry.operation;
+            mem_size  = entry.mem_size;
+            name      = entry.name.c_str();
+            function  = entry.function;
+		    return;
```

Comment:
We do not use tabs for indentation.

Reply:
I thought that I corrected this in the last commit